charts/monitoring-config/dashboards/graphql-stats.json

```diff
@@ -1490,12 +1490,12 @@
           "disableTextWrap": false,
           "editorMode": "code",
           "exemplar": true,
-          "expr": "sum by(job) (increase(http_requests_total{namespace=\"apps\", job=~\"publishing-api-read-replica\", controller=\"graphql\", action=\"execute\"}[1h]))",
+          "expr": "sum by(document_type) (increase(http_requests_total{namespace=\"apps\", job=~\"publishing-api-read-replica\", controller=\"graphql\", action=\"execute\"}[1h]))",
```

Comment:
I think this should probably be schema name rather than document type so that news articles are lumped together